### PR TITLE
Fix/ADF-864/Separate simple lists and remote lists creation

### PR DESCRIPTION
**Pull request by shpran**
# [ADF-864](https://oat-sa.atlassian.net/browse/ADF-864)

### CODE REVIEW

File: views/js/controller/list/index.js [hunk 1]
```diff
@@ -258,58 +274,103 @@ define([
...
-         */
+        // The list controller entrypoint
         start() {
             $('.form-submitter').off('click').on('click', (function (e) {
                 e.preventDefault();
-                handleCreateList();
+
+                const $form = $(e.target).closest('form');
+
+                urlUtil.route('remote', 'Lists', 'taoBackOffice').includes($form.attr('action'))
```
Comment (gabrielfs7) [inline views/js/controller/list/index.js:367]: I would move it to a function to explain better what is checking.

File: views/js/controller/list/index.js [hunk 2]
```diff
@@ -258,58 +274,103 @@ define([
...
-        /**
-         * The list controller entrypoint
-         */
+        // The list controller entrypoint
         start() {
             $('.form-submitter').off('click').on('click', (function (e) {
                 e.preventDefault();
-                handleCreateList();
+
+                const $form = $(e.target).closest('form');
```
Comment (gabrielfs7) [inline views/js/controller/list/index.js:365]: Is there a way to get the form by an identifier instead?

**Review by gabrielfs7 — APPROVED** [review]
Good job! Lets please validate with FE too :) 

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: views/js/controller/list/index.js (hunk 1 above)
Comment (bziondik) [reply views/js/controller/list/index.js:367]:
```suggestion
                $form.attr('action').includes('remote')
```

Comment (bziondik) [reply views/js/controller/list/index.js:367]: checking that action url contains `remote` substring

**Review by bziondik — APPROVED** [review]
`views/js/controller/list/index.js` should be refactored in future (move markup to tpl files, create generic functions)
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ]  New code is respecting code style rules
- [ ]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**